.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: "Build"
+defaults:
+  run:
+    shell: bash
+env:
+  DOCKER_REGISTRY: hazelops
```

Comment:
Depending on the use case of dockerizing this CLI, I may prefer to populate this into my docker registry for open source and public use, mostly for the matching username. Thoughts?